src/Tribe/Admin/Notice/Php_Version.php

```diff
@@ -92,18 +92,14 @@ public function should_display() {
 	 * @return string
 	 */
 	public function display_notice() {
+		/* Translators: %1$s list of plugins, %2$s current PHP version, %3$s link for read more */
+		$text = __( 'Starting February 2023, %1$s will require PHP 7.4 or later. Currently, your site is using PHP version %2$s. Please update to a newer version. %3$s.', 'tribe-common' );
 
-		// for PHP version above 5.4 and up to 5.6
-		if ( version_compare( phpversion(), '5.4.0' ) >= 0 ) {
-			$text = __( 'Starting March 2019, %1$s will no longer support versions prior to PHP 5.6. Your site is currently using PHP version %2$s which will no longer be supported by %1$s. For best results, we recommend using PHP 5.6 or above.', 'tribe-common' );
-		} else {
-			// for PHPversions below 5.4
-			$text = __( 'Starting March 2019, %1$s will no longer work with versions prior to PHP 5.4. Currently your site is using PHP version %2$s. For best results, we recommend using PHP 5.6 or above.', 'tribe-common' );
-		}
+		/* Translators: %1$s read more label */
+		$link = sprintf( '<a href="https://wordpress.org/support/update-php/" target="_blank">%1$s</a>', __( 'Read more.', 'tribe-common' ) );
```

Comment:
This could be part of the above (`$text`) string - why separate them?

Reply:
This way, we avoid the Problem of people messing with the HTML that we want.